src/EnergyPlus/MixedAir.cc

```diff
@@ -3279,6 +3276,155 @@ namespace MixedAir {
 						SetupEMSActuator( "Outdoor Air Controller", OAController( OAControllerLoop ).Name, "Air Mass Flow Rate", "[kg/s]", OAController( OAControllerLoop ).EMSOverrideOARate, OAController( OAControllerLoop ).EMSOARateValue );
 					}
 
+					VentMechObjectNum = OAController(OAControllerLoop).VentMechObjectNum;
```

Comment:
@mjwitte Thanks for finding the problem. I was just running out the door and decided to push the changes to the CI just in case I was lucky and everything worked. I will be putting up that fix next.